index.rst

```diff
@@ -49,6 +49,12 @@ Need help?
 
 The documentation above should answer most questions, but if you do need further help, consider checking out our `FAQ for Ubuntu Pro <https://discourse.ubuntu.com/t/ubuntu-pro-faq/34042>`_.
 
+Project and community
+=====================
+
+Ubuntu Pro Client is a member of the Ubuntu family. It’s an open source project that warmly welcomes community projects, contributions, suggestions, fixes and constructive feedback.
```

Comment:
```suggestion
Ubuntu Pro is a member of the Ubuntu family. It’s an open source project that warmly welcomes community projects, contributions, suggestions, fixes and constructive feedback.
```

Reply:
Just one item that I'm not sure is deliberate or not - the Project and community section refers to the Pro client rather than just Pro.